app/models/template.rb

```diff
@@ -87,6 +87,7 @@ def import_without_save(text, options = {})
     self.snippet = !!@importing_metadata[:snippet]
     self.default = options[:default] unless options[:default].nil?
     self.description = @importing_metadata[:description]
+    self.locked = options[:locked] unless options[:locked].nil?
```

Comment:
the plugin should use `:lock`, not `:locked`, then it would be handled by the line below I believe